onyo/commands/tests/test_unset.py

```diff
@@ -30,6 +30,9 @@
 contents: List[List[str]] = [[x, content_str] for x in assets]
 
 
+pytest.skip("GET not currently implemented", allow_module_level=True)
```

Comment:
Is it right, that the message in `test_unset.py` does states "GET not currently implemented"?